.travis.settings.xml

```diff
@@ -1,4 +1,12 @@
 <settings>
+    <mirrors xmlns="http://maven.apache.org/SETTINGS/1.1.0">
```

Comment:
Is `https://` a valid protocol here, too? If yes, let's use it.

Reply:
IIRC the namespace URI is *only* a String that is used for namespacing XML tags and attributes. It is not a link and therefore https is not important here and  not really a protocol but just part of the name. I think changing the namespace from "http://" to "https://" changes the namespace (but not cause any problem because it is not important). Therefore I would like to keep it like it is.

Reply:
Alright. Marking this conversation as resolved.